contracts/library/UniswapRouter.sol

```diff
@@ -0,0 +1,98 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.4;
+
+import "hardhat/console.sol";
+import "../ubeswap/contracts/uniswapv2/interfaces/IUniswapV2Router02.sol";
+import "../ubeswap/contracts/uniswapv2/interfaces/IUniswapV2Router02SwapOnly.sol";
+import "../openzeppelin-solidity/contracts/SafeERC20.sol";
+
+/**
+ * UniswapRouter is a library containing helper functions for interacting with contracts implementing
+ * the IUniswapV2Router02 interface, or subsets of its functionality.
+ **/
+library UniswapRouter {
```

Comment:
this seems like a great use case for a library. probably reduces the contract size too